lib/i18n/taxes.js

```diff
@@ -0,0 +1,31 @@
+const { defineMessages } = require('react-intl');
+
+const typeMsg = defineMessages({
+  VAT: {
+    id: 'tax.vat',
+    defaultMessage: 'Value-added tax (VAT)',
```

Comment:
I think the lang files are not reflecting these updates.

Reply:
Good catch! I had an invalid `require` (instead of `import`) there that the translations script didn't seem to like